apps/website/.vuepress/sidebar.js

```diff
@@ -105,6 +105,10 @@ module.exports = [
         title: 'Overview',
         path: '/releases/',
       },
+      {
```

Comment:
Note, this site was already published but I also need to merge this change or it will get reverted during the next release. .